lib/dependencies/ContextDependencyHelpers.js

```diff
@@ -5,19 +5,43 @@
 var ContextDependencyHelpers = exports;
 
 ContextDependencyHelpers.create = function(Dep, range, param, expr, options) {
```

Comment:
It looks like adding a commonjs `require` to harmony module file prevents `__webpack_require__.e` from being defined, crashing at runtime in the `Array.map` call in `webpackAsyncContext`. I'll file that under "don't do that" (for now) and split the tests :)